public/app/features/alerting/unified/state/AlertingQueryRunner.ts

```diff
@@ -241,3 +285,14 @@ const applyChange = (
 
   return nextResult;
 };
+
+const createErrorPanelData = (error: unknown): PanelData => ({
+  series: [],
+  state: LoadingState.Error,
+  errors: [
+    {
+      message: stringifyErrorLike(error),
```

Comment:
The default error messages from `dag` are quite rough:
> cannot link input node named G since it doesn't exist in graph

I wonder if, by looking at the error details, we could provide more user-friendly message

Reply:
How do we feel about something like this?
<img width="549" alt="image" src="https://github.com/user-attachments/assets/5b9476cd-1c25-4476-959a-3a184eea2323" />

Reply:
Awesome!